api/src/logger.ts

```diff
@@ -44,6 +44,30 @@ if (env['LOG_STYLE'] !== 'raw') {
 		},
 	};
 }
+if (env['LOG_STYLE'] === 'raw') {
```

Comment:
Should this be an `else` or perhaps inverting the above `!== 'raw'` block then adding the `else`?

Reply:
Considered that but found this "explicit if" more readable.